src/pages/Preferences.tsx

```diff
@@ -22,41 +22,51 @@ export const Preferences: React.FC = () => {
   >(GET_USER_SETTINGS);
   usePageTitle("Preferences");
   const userSettings = get(data, "userSettings");
-  if (tab === undefined) {
-    return <Redirect to={`${paths.preferences}/profile`} />;
+
+  if (!tabRouteValues.includes(tab as PreferencesTabRoutes)) {
```

Comment:
if you set change `const { tab } = useParams<{ tab: string }>();` above to `const { tab } = useParams<{ tab: PreferencesTabRoutes }>();` then does it work / remove the need to use `tab as PreferencesTabRoutes` here?

Reply:
I tried this and `tab` is still type string